tests/test_header.py

```diff
@@ -33,3 +43,37 @@ def is_linked_fixture(self, request):
         sectPr_cxml, expected_value = request.param
         header = Header(element(sectPr_cxml), None, WD_HEADER_FOOTER.PRIMARY)
         return header, expected_value
+
+    @pytest.fixture
+    def body_fixture(self, part_prop_, block_item_):
+        # this is what i want to do but it throws an error...
+        # > sectPr = element('w:sectPr/w:headerReference{w:type=default,r:id=rId8}')
+        # < lxml.etree.XMLSyntaxError: Namespace prefix r for id on headerReference is not defined, line 2, column 50
+        # how do i fix?
```

Comment:
@scanny Ok cool. I updated the commit. Much cleaner using that `CT_HdrFtrRef`.

I took the name from the analysis doc. I forget why we named it that but I think we probably had a good reason. Maybe.

Adding the `CT_HdrFtrRef` didn't fix the `r:id` cxml issue. Want to take a look?